31354/change structure for inputs

jwgoedert opened the pull request:
<!--
Hi there! Thanks for contributing to WordPress!

Pull Requests in this GitHub repository **must** be linked to a ticket in the WordPress Core Trac instance (https://core.trac.wordpress.org), and are only used for code review. **No pull requests will be merged on GitHub.**

See the WordPress Handbook page on using PRs for Code Review more information: https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/

If this is your first time contributing, you may also find reviewing these guides first to be helpful:
- FAQs for New Contributors: https://make.wordpress.org/core/handbook/tutorials/faq-for-new-contributors/
- Contributing with Code Guide: https://make.wordpress.org/core/handbook/contribute/
- WordPress Coding Standards: https://make.wordpress.org/core/handbook/best-practices/coding-standards/
- Inline Documentation Standards: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/
- Browser Support Policies: https://make.wordpress.org/core/handbook/best-practices/browser-support/
- Proper spelling and grammar related best practices: https://make.wordpress.org/core/handbook/best-practices/spelling/
-->

<!-- Insert a description of your changes here -->
Nested child elements in list items to create visual hierarchy, moved input fields to end of labels for screen reader usability 
Trac ticket: <!-- insert a link to the WordPress Trac ticket here -->
https://core.trac.wordpress.org/attachment/ticket/31354/

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

jwgoedert committed:
Add class and selectors for indenting children

jwgoedert committed:
Apply 31354.5.patch remove changes for disabling fields

jwgoedert committed:
Remove empty php tags from disable variable removal

joedolson commented on src/wp-admin/options-discussion.php:
```diff
@@ -39,10 +39,14 @@
 <h1><?php echo esc_html( $title ); ?></h1>
 
+<!-- ARIA live region for screen readers -->
+<div id="aria-live-region" aria-live="polite" class="screen-reader-text"></div>
```
Remove this; it's a stray leftover from your previous patch.

joedolson reviewed (changes requested):
Some JS-related code left in.

jwgoedert committed:
Remove leftover aria-live js code

jwgoedert reviewed (commented):
Removed leftover js code.

joedolson committed:
Text & formatting updates

Remove extraneous `br`, extraneous line breaks, change close comments text, word choice change.

joedolson commented:
in r59160

joedolson commented:
Issue was closed in https://core.trac.wordpress.org/changeset/59160
